src/partitionGenerator.cpp

```diff
@@ -479,7 +491,7 @@ int main(int argc, char* argv[])
     global_nexus_collection->link_features_from_property(nullptr, &link_key);
     // make a global network
     Network global_network(global_nexus_collection);
-
+    std::cout<<"HERERERERE\n";
```

Comment:
I suspect this `cout` statement was intended to be removed.

Reply:
Yes, I'll fix that quick.